rust/candid/src/parser/grammar.lalrpop

```diff
@@ -81,6 +81,7 @@ pub Arg: IDLValue = {
     },
     "variant" "{" <VariantField> "}" => IDLValue::Variant(Box::new(<>), 0),
     "principal" <Text> =>? Ok(IDLValue::Principal(Principal::from_text(<>).map_err(error)?)),
+    AnnVal => <>,
```

Comment:
From the spec, you also need to update line 51 to use `AnnVal` for `IDLArgs`, so that you can avoid the top level parenthesis.

Reply:
> so that you can avoid the top level parenthesis

You mean this production in the spec, right?
```
<arg> ::= ( <annval>,* )
```
These parens don't look like they're optional. Are they supposed to be optional?

Reply:
Yes, the parens inside `annval` is optional. You can say `(42 : int, 42 : nat)`, `((42 : int), (42 : nat))` is not required.